en/faq/postcss-plugins.md

```diff
@@ -5,7 +5,20 @@ description: How to add PostCSS plugins?
 
 # How to add PostCSS plugins?
 
-In your `nuxt.config.js` file:
+### Default Method
```

Comment:
I'd probably switch the two heading. The "default" should be using `nuxt.config.js`. The "legacy" or "traditional" way is a postcss.config.js

Reply:
@manniL will it good if I also swap two methods? Then the first will be default (nuxt.config.js) and the second (postcss.config.js) ?

Reply:
Yes, please. :relaxed: